src/components/TrialBanner.js

```diff
@@ -69,14 +69,14 @@ export const TrialBanner = Utils.connectStore(authStore, 'authState')(class Tria
       div({
         style: {
           display: 'flex', alignItems: 'center', padding: '1.5rem', height: 95,
-          backgroundColor: isWarning ? colors.warning() : '#359448',
+          backgroundColor: isWarning ? colors.warning() : '#28873b',
           justifyContent: 'center', color: 'white', width: '100%', fontSize: '1rem'
         }
       },
       [
         div({
           style: {
-            fontSize: '1.5rem', fontWeight: 500, textAlign: 'right', borderRight: '1px solid', paddingRight: '1rem', marginRight: '1rem',
```

Comment:
Why not just increase the `maxWidth`?

Reply:
Oh yes that seems better 👍will make that change